comm.h

```diff
@@ -22,86 +22,46 @@
 #include <errno.h>
```

Comment:
Commenting on file: not sure if it's really necessary - or if these structures could live on `endpoint.cpp`, only place they are really used.